src/wallet/wallet.cpp

```diff
@@ -922,6 +922,37 @@ bool CWallet::MarkReplaced(const uint256& originalHash, const uint256& newHash)
     return success;
 }
 
+void CWallet::SetDirtyState(const uint256& hash, unsigned int n, bool dirty)
+{
+    const CWalletTx* srctx = GetWalletTx(hash);
+    if (srctx) {
+        CTxDestination dst;
+        if (ExtractDestination(srctx->tx->vout[n].scriptPubKey, dst)) {
+            AssertLockHeld(cs_wallet);
```

Comment:
Commit "wallet: enable avoidreuse feature"

Why is this here? Move up?

Same in `CWallet::IsDirty` below.

Reply:
This may be artifact from previous iteration. Removing.